app/src/androidTest/java/de/test/antennapod/ui/BatchFeedSettingsTest.java

```diff
@@ -0,0 +1,190 @@
+        feeds.get(2).setPreferences((defaultFeedPreferences));
+        feeds.add(uiTestUtils.hostedFeeds.get(3));
+        feeds.get(3).setPreferences((defaultFeedPreferences));
+
+        Intent intent = new Intent();
+//        intent.putExtra(MainActivity.EXTRA_FEED_ID, feeds.getId());
+        EspressoTestUtils.setLastNavFragment(SubscriptionFragment.TAG);
+
+        activityRule.launchActivity(intent);
+        activityRule.getActivity().loadChildFragment(BatchFeedSettingsFragment.newInstance(feeds));
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        uiTestUtils.tearDown();
+    }
+
+    @Test
+    public void testClick_keepUpdatedPreference() {
+        try {
+            Thread.sleep(2000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+
+        FeedPreferences expectedFeedPreferences = new FeedPreferences(0, true, FeedPreferences.AutoDeleteAction.GLOBAL,
+                VolumeAdaptionSetting.OFF, null, null);
+        expectedFeedPreferences.setKeepUpdated(false);
+//        clickPreference(R.string.keep_updated);
+        onView(withId(R.id.recycler_view)).perform(RecyclerViewActions.actionOnItemAtPosition(0, new CustomViewAction()));
+
+        Awaitility.await().atMost(4000, MILLISECONDS)
+                .until(() -> {
+                    for (Feed feed : feeds) {
+                        Assert.assertEquals(expectedFeedPreferences.getKeepUpdated(), feed.getPreferences().getKeepUpdated());
+                    }
+                    return true;
+                });
+
+        expectedFeedPreferences.setKeepUpdated(true);
+        onView(withId(R.id.recycler_view)).perform(RecyclerViewActions.actionOnItemAtPosition(0, new CustomViewAction()));
+
+        Awaitility.await().atMost(4000, MILLISECONDS)
+                .until(() -> {
+                    for (Feed feed : feeds) {
+                        Assert.assertEquals(expectedFeedPreferences.getKeepUpdated(), feed.getPreferences().getKeepUpdated());
+                    }
+                    return true;
+                });
+        try {
+            Thread.sleep(1000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+//        clickPreference(R.string.authentication_label);
+//        onView(withText(R.string.cancel_label)).perform(click());
+//
+//        clickPreference(R.string.playback_speed);
+//        onView(withText(R.string.cancel_label)).perform(click());
+//
+//        clickPreference(R.string.pref_feed_skip);
+//        onView(withText(R.string.cancel_label)).perform(click());
+//
+//        clickPreference(R.string.auto_delete_label);
+//        onView(withText(R.string.cancel_label)).perform(click());
+//
+//        clickPreference(R.string.feed_volume_reduction);
+//        onView(withText(R.string.cancel_label)).perform(click());
+    }
+
+
+    @Test
+    public void testClick_playBackSpeed() {
+        try {
+            Thread.sleep(2000);
```

Comment:
Please don't manually sleep in tests. This makes the tests flaky and slow. Instead, wait for a specific condition using Awaitility.